SocketRocket/SRWebSocket.m

```diff
@@ -570,6 +570,33 @@ - (void)_initializeStreams;
     
     _inputStream.delegate = self;
     _outputStream.delegate = self;
+    
+    NSString *networkServiceType;
+    switch (_urlRequest.networkServiceType) {
+        case NSURLNetworkServiceTypeDefault:
+            return;
```

Comment:
I'd rather us not early `return` out of this. A break here seems like it would be fine. Having returns here could lead to undesired behaviors if somebody adds initialization logic to the end of this function.

My recommendation would be to move this into its own function called `initializeNetworkServiceType:` and inside that you can use early returns.